### ISSUE TITLE
Search for gene list

### ISSUE THREAD

**Issue by YuanzhongChang** [lxl0928/ricedata_spider#1]
I get a list of DE gene for my study. I want to get annotation for them by ricedata. The most I want to get is the known genes in my list.

**Comment by lxl0928** [comment]
Yep, you can  cantact me by the website  [https://blog.timilong.com/about](https://blog.timilong.com/about)
